build.gradle.kts

```diff
@@ -188,15 +188,15 @@ tasks {
         // causes errors on JDK 16 due to usage of internal JDK APIs. See this
         // [blog post](https://melix.github.io/blog/2021/03/gradle-java16.html)
         // for details.
-        options.setIncremental(false)
+        options.isIncremental = false
```

Comment:
Gradle 7 has been released in the meantime. So updating to it and removing this option would be the best solution.